raster-foundry/raster-foundry
# Add redirect to a connection specified in BUILDCONFIG

## Overview
Add the options to redirect to a social connection login from the login page
Allow creating tokens on redirected connection

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] [Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~
~- [ ] Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~

### Notes


## Testing Instructions

 * add a redirect connecton to your overrides file:
```
const REDIRECT_CONNECTION = JSON.stringify({
    clientId: '544Cz1V8tqezMCbyN8mNJmlzOpd8H4ex',
    connection: 'TestClient'
});
module.exports = function(_path){
...
new webpack.DefinePlugin({
  'BUILDCONFIG': {
    ...
    REDIRECT_CONNECTION
  }
}
```
* Verify that you can login using the social connection
* Verify that you can create an api token on the social connection account

Closes https://github.com/azavea/raster-foundry-platform/issues/297
